Dockerfile.prod

```diff
@@ -0,0 +1,28 @@
+FROM python:3.12.7-alpine3.20@sha256:edd1d8559c585e1e9a9b79de44ac27f8ac32cb0c7323e112ae6870ceeecd8dbf AS builder
+
+COPY requirements.txt ./
+
+RUN set -eux; \
+    \
+    # Install pip dependencies \
+    python3 -m pip install --no-cache-dir -r requirements.txt; 
+
+FROM python:3.12.7-alpine3.20@sha256:edd1d8559c585e1e9a9b79de44ac27f8ac32cb0c7323e112ae6870ceeecd8dbf
+
+WORKDIR /object-storage-api-run
+
+COPY --from=builder /usr/local/lib/python3.12/site-packages /usr/local/lib/python3.12/site-packages
+COPY --from=builder /usr/local/bin /usr/local/bin
+
+COPY README.md ./
+COPY object_storage_api/ object_storage_api/
+
+RUN set -eux; \
+    \
+    # Create loging.ini from its .example file \
+    cp object_storage_api/logging.example.ini object_storage_api/logging.ini;
+
+USER nobody
+
+CMD ["uvicorn", "object_storage_api.main:app", "--app-dir", "/object-storage-api-run", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
We recently started using the [FastAPI CLI](https://fastapi.tiangolo.com/fastapi-cli/) CLI command line program, which internally uses a production-ready Uvicorn server, as that now seems to be the recommended way by FastAPI of running apps for, both, local development and production.